src/auth/auth.guard.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { ExecutionContext, UnauthorizedException } from '@nestjs/common';
```

Comment:
Specs for `AuthGuard` which is used to validate if an `x-userid` header is present in the request.